master/buildbot/process/buildstep.py

```diff
@@ -511,6 +511,9 @@ def setRenderable(res, attr):
             if self.results != CANCELLED:
                 self.results = EXCEPTION
 
+        self.updateSummary()
+        yield self.updateSummary.flush()
```

Comment:
This fails with error: `builtins.AttributeError: 'function' object has no attribute 'flush'`

But this works: `yield self.realUpdateSummary.flush() `